subsys/infinispan/src/main/resources/infinispan.xml

```diff
@@ -116,5 +115,22 @@
       </persistence>
     </local-cache>
 
+    <!-- TODO: these should be a distributed, non-persistence cache when enable cluster -->
+    <local-cache name="store-data" configuration="local-template">
```

Comment:
This is cool here, but for staging / production it's probably not safe to leave it on the pod. We should just persist directly to JDBC immediately when we deploy this code out there, so we can start looking at client tooling to help us manage repo definitions.

Reply:
Agree. This needs to be persisted to jdbc ATM for prod/stage.